include/acl_selectors.php

```diff
@@ -364,7 +375,7 @@ function populate_acl($user = null, $show_jotnets = false) {
 		'$aclModalTitle' => t('Permissions'),
 		'$aclModalDismiss' => t('Close'),
 		'$features' => array(
-		"aclautomention"=>(feature_enabled($user['uid'],"aclautomention")?"true":"false")
+		'aclautomention' => (feature_enabled($user['uid'],"aclautomention")?"true":"false")
```

Comment:
Standards: Please add a space after commas and spaces around operators like `?` and `:`.